src/css/App.scss

```diff
@@ -288,6 +305,7 @@ label {
 }
 
 .coa-page_content {
+  .coa-overlay__content,
```

Comment:
This file is getting way to huge. Can we start to break it up into smaller .scss files plz? 🙏

Reply:
YASSSSS, I was reticent to pick a structure without your input. I'll take a stab at it in my next PR so this one doesn't get unruly.